DQM/Physics/test/ExoticaDQM_cfg.py

```diff
@@ -1,31 +1,92 @@
 import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.VarParsing import VarParsing
+
+
+## VarParsing object
+
+options = VarParsing('analysis')
+
+options.register('globaltag', '122X_mcRun3_2021_realistic_v1', VarParsing.multiplicity.singleton, VarParsing.varType.string, 'Set Global Tag')
+options.register('name', 'TEST', VarParsing.multiplicity.singleton, VarParsing.varType.string, 'Set process name')
+
+options.parseArguments()
+
+
+## Process
+
 process = cms.Process("ExoticaDQM")
 
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 process.load("Configuration.StandardSequences.MagneticField_cff")
-process.GlobalTag.globaltag = cms.string("80X_mcRun2_asymptotic_v13")
+process.GlobalTag.globaltag = cms.string(options.globaltag)
 
 process.load("DQM.Physics.ExoticaDQM_cfi")
 process.load("DQMServices.Core.DQM_cfg")
 process.load("DQMServices.Components.DQMEnvironment_cfi")
 
-process.dqmSaver.workflow = cms.untracked.string('/Physics/Exotica/TEST')
+
+process.load('DQMOffline.Configuration.DQMOffline_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+
+process.DQMStore.verbose = cms.untracked.int32(2)
```

Comment:
Please remove here and below the type, i.e.
process.DQMStore.verbose = 2
following:
https://cms-sw.github.io/cms_coding_rules.html#6--packaging-rules-1

Reply:
I couldn't remove the type on process.DQMStore.verbose since I got the following error message when trying to run:
`Exception Message:`
` unknown python problem occurred.`
`TypeError: verbose does not already exist, so it can only be set to a CMS python configuration type`

I checked that the setting of this parameter does not affect the cmsRun output so I dropped it.
Other types were removed successfully.